tests/cases/fourslash/incompleteFunctionCallCodefixTypeParameterVariable.ts

```diff
@@ -19,24 +19,24 @@ verify.codeFix({
   added2(et)
 }
 
-function added1(et: string) {
+function added1<T extends "phone" | "home">(et: T) {
```

Comment:
I believe this was the original true intention here but the type widened since it had a union constraint. This was introduced in https://github.com/microsoft/TypeScript/pull/49727 , cc @JoshuaKGoldberg